service/category/category_service_impl_test.go

```diff
@@ -83,6 +84,39 @@ func TestGetAll(t *testing.T) {
 				).Once()
 			},
 		},
+		{
+			name:          "it should return valid categories, when repository return nil error",
```

Comment:
@SariFatur 

Line 465 ganti ke ini sar: `it should return service.ErrRepository, when category repository return a repository.ErrDatabase error`

Sama line 488: `it should return service.ErrRepository, when thread repository return a repository.ErrDatabase error`

Habis itu kamu commit & push ulang aja, ga perlu di pull request